kerl

```diff
@@ -2099,6 +2099,39 @@ get_active_path() {
     fi
 }
 
+fix_otp_installations() {
+    if [ -f "$KERL_BASE_DIR"/otp_installations ]; then
+        inexisting_names=""
```

Comment:
Gonna point out that "inexisting" isn't a word (well, actually it is, but it's obsolete and doesn't mean what you think it means). You want "nonexistant" or possibly just "missing".

Reply:
Cool. Naming, right? (I guess I "translated" directly from portuguese "inexistente") Thanks. I'll change it.

Reply:
f9c546a